tests/markov/msm/test_msm.py

```diff
@@ -289,12 +289,16 @@ def test_score(self):
         self._score(self.msm_sparse)
 
     def _score_cv(self, estimator):
-        s1 = score_cv(estimator, self.dtraj, lagtime=10, n=5, score_method='VAMP1', score_k=2).mean()
+        def fit_fetch(dtrajs):
+            cc = TransitionCountEstimator(lagtime=10, count_mode="sliding").fit(dtrajs).fetch_model().submodel_largest()
```

Comment:
why's it called `cc`?

Reply:
I guess it should be called `cm` or even better `count_model`, thanks